verify.js

```diff
@@ -0,0 +1,28 @@
+const db = require('.');
+
+const noDuplicates = Object.keys(db)
+  .filter(key => key !== 'schemas')
+  .map(list => {
+    let numberOfItems;
+    let numberOfUniques;
+    if (list == 'companies') {
+      let companyCodes = db[list].map(company => company.code);
+      numberOfItems = companyCodes.length;
+      numberOfUniques = (new Set(companyCodes)).size;  
+    } else {
+      // Service(s), Characteristic(s) and Descriptor(s)
+      let uuids = db[list].map(gatt => gatt.uuid);
+      numberOfItems = uuids.length;
+      numberOfUniques = (new Set(uuids)).size;
+    }
+
+    if (numberOfItems != numberOfUniques) {            
+      console.error(`Failed to verify '${list}':`);
+      console.error(`'${list}' contains '${numberOfItems - numberOfUniques}' duplicates.`);
```

Comment:
You might want to combine the two `console.error`s into a single one for nicer output, since both logs stacktrace which will looks awkward.